pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -842,7 +842,7 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 // This entails adding required, missing SecurityRules and removing stale rules.
 func (az *Cloud) reconcileSecurityGroup(clusterName string, service *v1.Service, lbIP *string, wantLb bool) (*network.SecurityGroup, error) {
 	serviceName := getServiceName(service)
-	glog.V(5).Infof("reconcileSecurityGroup(%s): START clusterName=%q lbName=%q", serviceName, clusterName)
+	glog.V(5).Infof("reconcileSecurityGroup(%s): START clusterName=%q", serviceName, clusterName)
```

Comment:
seems good in the context of this PR.
if the load balancer name is _really_ wanted here it would need a small refactor.

/cc @justaugustus
/cc @kubernetes/sig-cloud-provider-pr-reviews